packages/elements/list/src/queries/marks/hasListItemContentMarkActive.tsx

```diff
@@ -0,0 +1,13 @@
+import { getNode } from '@udecode/plate-common';
+import { TEditor, TNode } from '@udecode/plate-core';
+import { ListItemContentSelection } from '../../types';
+
+export const hasListItemContentMarkActive = (
```

Comment:
I checked for the namings and this is the same as `getMark` so I renamed it accordingly `getListItemContentMark` since I thing this might be useful is some cases when it comes to toolbar elements